data_doc_template.Rmd

```diff
@@ -0,0 +1,65 @@
+---
+output: html_document
+params:
+  data_name: "hamilton_landcover"
+  data_version: "0.1.0"
+---
+
+```{r, echo = FALSE}
+library(CODECtools)
+library(knitr)
+options(knitr.kable.NA = '')
+
+d <- read_tdr_csv(glue::glue("https://github.com/geomarker-io/{params$data_name}/releases/download/v{params$data_version}")) |>
+  dplyr::rename(census_tract_id_2010 = census_tract_id)
+
+#### should we do check here? 
+
+# write_tdr_csv(hamilton_landcover)
+# check_codec_tdr_csv("hamilton_landcover")
+```
+
+
+# `r glimpse_attr(d)$value[glimpse_attr(d)$name == "title"]`
+
+**last updated:** should we make this part of the metadata? 
```

Comment:
We could use R to check the release date on github associated with the version, but this would only apply to GitHub data pipelines like we have for CoDEC.

This would get you the date the *latest* release was published, since the API returns all releases, with the most recent being the first in the list.

```r
gh::gh("/repos/geomarker-io/hamilton_landcover/releases")[[1]]$published_at
```